k8s/simple-akka-cluster-deployment.yml

```diff
@@ -28,6 +28,8 @@ spec:
               fieldRef:
                 apiVersion: v1
                 fieldPath: status.podIP
+          - name: "DYN_JAVA_OPTS"
+            value: "-Dakka.remote.netty.tcp.hostname=${HOSTNAME//./-}.default.pod.cluster.local -Dakka.management.http.hostname=${HOSTNAME//./-}.default.pod.cluster.local"
```

Comment:
Is the replace necessary? Is it even possible to have dot in the pod hostname? I think if you try to create a pod with dot within the name you get the error: `a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character`

Reply:
@gkocur - It is actually the IP address which has dots, which is being replaced by hyphens (so `127.0.0.1` would become `127-0-0-1`..). `HOSTNAME` env variable is infact set in the [previous line](https://github.com/softwaremill/akka-simple-cluster-k8s/pull/2/files/086c1be5badc066c39bcdedee982ea006b748b1c#diff-126b400e2c9422fae43c8e2f4431fedcR30) just above this one. This way of translation of pod IP to pod's dns is [defined by kubernetes](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#a-records-1).